packages/cmatrix.nix

```diff
@@ -0,0 +1,26 @@
+#This is the toolbox containing the things we need to build and operate a PKG
+{
+  stdenv,
+  fetchFromGitHub,
+  ncurses,
+  autoreconfHook,
+}:
+#This is the bill of materials and instructions
+stdenv.mkDerivation {
+  pname = "cmatrix";
+  version = "2.0";
+  #This is the tools we need to build
+ nativeBuildInputs = [
+ autoreconfHook
+ ];
+ buildInputs = [
```

Comment:
Indentation looks wonky here, try running the formatter

Reply:
elbows